modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java

```diff
@@ -60,12 +59,12 @@ public Map<String, Object> getParameters(
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
 
 		return HashMapBuilder.<String, Object>put(
-			"firstDayOfWeek", _getFirstDayOfWeek()
+			"firstDayOfWeek", _getFirstDayOfWeek(ddmFormFieldRenderingContext)
```

Comment:
Here, you could replace to: `dmFormFieldRenderingContext.getLocale()`.

Reply:
Done

Reply:
Done